css-canvas-medical/styles.css

```diff
@@ -0,0 +1,220 @@
+body {
+  width: 1150px;
+  margin: auto;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+}
+
+.line-height {
+  line-height: 35px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-header {
+  padding-top: 30px;
+}
+
+h1 {
+  color: #284564;
+  font-family: "Tiempos Headline", sans-serif;
+  font-weight: 300;
+  letter-spacing: 3px;
+}
+
+h2 {
+  font-family: "Tiempos Headline", sans-serif;
+  font-size: 30px;
+  color: white;
+  letter-spacing: 1px;
+  font-weight: lighter;
+}
+
+.column-two-thirds p {
+  color: white;
+  line-height: 25px;
+  font-family: Avenir, sans-serif;
+}
+
+.text-left {
+  text-align: left;
+}
+
+.header-links li {
+  display: inline;
+  padding: 20px;
+  color: #b6b6b6;
+  font-family: Avenir, sans-serif;
+}
+
+.inline-row {
+  justify-content: space-between;
+  border-bottom: solid rgb(244 244 246);
+}
+
+button {
+  background-color: #284564;
+  color: white;
+  border-radius: 5px;
+  border-style: solid;
+  border-color: #284564;
+  font-family: Avenir, sans-serif;
+}
+
+.contact-us-btn {
+  height: 40px;
+  width: 90px;
+}
+
+img {
+  width: 100%;
+}
+
+.hero {
+  position: relative;
+  margin-top: 60px;
+}
+
+.hero-image {
+  background-image: url("images/hero.jpg");
+  background-size: cover;
+  height: 500px;
+  width: 1000px;
+}
+
+.column-two-thirds {
+  position: absolute;
+  top: 30px;
+  left: 70px;
+  width: 40%;
+}
```

Comment:
The only valid property applied to a `column-` class is `flex-basis`. Everything else belongs on a separate class that you can add to the element in addition to the `column-` layout class. This issue is present in multiple other places in your CSS as well, please make sure you address it wherever it is happening.